src/parser.rs

```diff
@@ -864,11 +864,21 @@ impl Parser {
             self.parse_create_view()
         } else if self.parse_keyword("EXTERNAL") {
             self.parse_create_external_table()
+        } else if self.parse_keyword("SCHEMA") {
+            self.parse_create_schema()
         } else {
-            self.expected("TABLE, VIEW or INDEX after CREATE", self.peek_token())
+            self.expected(
+                "TABLE, VIEW, INDEX or SCHEMA after CREATE",
```

Comment:
I think with the next addition it makes sense to change the message, so that it doesn't duplicate the list above (as it's incorrect already, missing "EXTERNAL").